### DISCUSSION TITLE
nothing but a Bad Gateway

### DISCUSSION THREAD

**Discussion by jtarchie** [weather-gov/api#544] [Q&A] [answered]
Cannot report issues anywhere, so sorry for the incorrect location.

When hitting the endpoint https://api.weather.gov/ and any other API endpoint, I'm getting a 502 Bad Gateway.
Happy to help debug from my end.

Thanks!

**Comment by roncnelson** [comment]
I'm get a 503 error on all requests.  HELP!!!

**Comment by JJKraw** [comment, marked as answer]
Report operational issues like this to nco.ops@noaa.gov

You will get put on an email thread so you can know how things are progressing. 

I believe this all has cleared up from yesterday.
